ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/as/webservices/main/module.xml

```diff
@@ -72,5 +72,8 @@
         <module name="io.undertow.servlet"/>
         <module name="org.wildfly.transaction.client"/>
         <module name="javax.transaction.api"/>
+        <module name="javax.xml.rpc.api"/>
```

Comment:
@parsharma This line was removed from this file in other work recently and I think this line probably back crept in during a merge or rebase or something. I don't see anything in the rest of the PR that would require it so I suspect it wasn't intentionally part of this commit.

I believe removing this will allow this PR to build. If you do that and the PR passes CI please ping me in chat so I can make sure it gets merged.